docs/go/how-to-develop-a-worker-program-in-go.md

```diff
@@ -85,6 +89,8 @@ import SharedWorkersTaskQueueRegistrationNote from '../reminders/note-workers-ta
 
 ### Register multiple Types
```

Comment:
Could change this to "Registering"... I think I went through an anti-gerund period - but took it took far with these sub titles.

Reply:
Updated